misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/evaluation/engine/BallerinaTypeResolver.java

```diff
@@ -73,7 +77,32 @@
         return resolvedTypes;
     }
 
+    /**
+     * @param context        debug context
+     * @param typeDescriptor type descriptor string
+     * @return a collection of resolved types
+     * @throws EvaluationException if unsupported type(s) found
```

Comment:
```suggestion
     * @throws {@link EvaluationException} if unsupported type(s) found
```

Reply:
@prabushi We can directly define Exception without the @link annotation as suggested in the [Oracle Javadocs documentation](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#throwstag) (Also the intellij gives a warning when using `@link` with `@throws`).